packages/core/integration-tests/test/cache.js

```diff
@@ -4287,7 +4270,7 @@ describe('cache', function() {
                   ?.filePath,
                 'utf8',
               );
-              assert.equal(html.match(/<script/g)?.length, 5);
+              assert.equal(html.match(/<script/g)?.length, 6);
```

Comment:
what changed here?

Reply:
Not sure. I assume the size of one of the bundles is different or something.